spec/unpoly/element_spec.coffee

```diff
@@ -204,6 +204,13 @@ describe 'up.element', ->
       results = up.element.subtree($element[0], '.match')
       expect(results).toEqual []
 
+  describe 'up.element.isInSubtree()', ->
+    it 'can handle conflicts with forms where an input is named `contains` GH#507', ->
+      $form = $fixture('form')
+      $child = $form.affix('input[name="contains"]')
+
+      expect(up.element.isInSubtree($form[0], $child)).toBe(true)
```

Comment:
I had to force this to be a `Node` to be able to use the prototype, but that have might exposed another issue where elements other than a `Node` would be passed as the `root`. Thoughts?

Reply:
> I had to forcefully grab the first element to be able to use the prototype function of Node

That's just because `$fixture()` returns a jQuery collection instead of a native `Element`. `$form[0]` returns the one native DOM element in that collection.

As an alternative we can use the `fixture()` and `up.element.affix()` helpers, which return an `Element`. We are eventually going to hose jQuery off the test suite.

> but that have might exposed another issue where elements other than a Node would be passed as the root

That's fine. We don't need to support arguments of any other type.